security/vulnerability-management.md

```diff
@@ -0,0 +1,92 @@
+design through deployment and maintenance.
+
+## 3. Definitions
+- **Vulnerability**: A weakness in the code that can be exploited to compromise the confidentiality, integrity, or 
+  availability of the system.
+- **Remediation**: The process of fixing a vulnerability.
+- **Waive**: Rule to ignore a vulnerability for a specific amount of time
+- **Severity Levels**: Classification of vulnerabilities based on their potential impact.
+
+## 4. Roles and Responsibilities
+- **Developers:** Responsible for writing secure code, conducting initial testing, and fixing identified vulnerabilities.
+- **Project Managers:** Responsible for ensuring that vulnerabilities are addressed and review waivers for 
+  vulnerabilities in accordance with this policy
+
+## 5. Classification of Vulnerabilities
+Vulnerabilities will be classified into four severity levels:
+- **Critical**: Immediate action required. These vulnerabilities can lead to complete system compromise, significant 
+  data breach/loss.
+- **High**: Must be addressed as soon as possible. These can lead to major security breaches.
+- **Medium**: Should be fixed within a reasonable timeframe. These can potentially be exploited but are not as severe.
+- **Low**: Minor issues that should be resolved in the regular course of development.
+
+## 6. Identification of Vulnerabilities
+The following mechanism will be used identify potential vulnerabilities
+- **IDE Tools**: Developer's IDE will be the first gate to identify potential vulnerabilities through static analysis 
+  or dependency scan
+- **Manual Code Reviews**: Developers will conduct regular manual code reviews to identify and raise any potential 
+  security concerns.
+- **Automated Scanning**: Automated tools will run on schedule or on commit actions, performing scans and triggering 
+  notifications
+- **External Audits**: Engage third-party security experts to conduct periodic security assessments.
+
+## 7. Remediation Process
+1. **Identification**: Identify the vulnerability and whether it impacts a public-facing application.
+2. **Assessment**: Assess the impact of the vulnerability if it were exploited and determine the severity.
+3. **Prioritization**: Prioritization of the investigation and potential remediation depending on the severity. In some 
+   cases the developers might be able to determine that their implementation is not exploitable instead of removing the 
+   vulnerability
+4. **Remediation Plan**: A remediation plan is developed, including a timeline for addressing the vulnerability. For 
+   critical vulnerabilities some actions might need to be taken (such as temporarily disable a service)
+5. **Fix Implementation**: Developers implement the fixes as per the remediation plan. For high and critical 
+   vulnerabilities, immediate action is taken and a hotfix release might be required.
+6. **Verification**: The security team verifies the fix to ensure the vulnerability is effectively mitigated.
```

Comment:
Do we need to define what the Security Team is?

Reply:
@AndyLewis1971 , thank you for the comment,

The security team was a valid notion that I took from one of my readings and I missed afterward. My idea here is that we can have a second validation that doesn't come from the developer fixing the issue, just to validate that the fix is sensible for our organization.

Bear in mind that the remediation plan might not fully define how the developer will fix the issue or they might have to solve it in some other way to get this in a timely manner.

Would it work to change that paragraph to 
"The fix will be peer-reviewed by project managers and/or developers to ensure the vulnerability is effectively mitigated."

Reply:
I think that's a good solution @jcalcines